src/kms_key_provider.rs

```diff
@@ -44,6 +45,7 @@ impl KMSKeyProvider {
         let config = Config::builder()
             .region(Region::new(region.into()))
             .credentials_provider(aws_creds)
+            .retry_config(RetryConfig::new().with_max_attempts(5))
```

Comment:
What's the default if not specified? No retries?

Reply:
It seems like the default is 3 attempts. I've got a feeling the error that is happening is a non-retryable error so I might have to write some custom logic after all.